connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/importer/ProcessDefinitionImporter.java

```diff
@@ -53,7 +53,11 @@ public ProcessDefinitionImporter(
   public synchronized void scheduleImport() {
     try {
       var result = search.query();
-      handleImportedDefinitions(result);
+      try {
+        handleImportedDefinitions(result);
+      } catch (Exception e) {
+        LOG.error("Failed to handle imported process definitions", e);
+      }
       ready = true;
     } catch (Exception e) {
       LOG.error("Failed to import process definitions", e);
```

Comment:
Perhaps this error message should be different now, since we now have an import process definition-specific error message on line 59

Maybe something like "Operate search query failed"?

Reply:
As discussed on Slack, the error messages already in this PR better reflect the difference between `search.query()` throwing an exception and `handleImportDefinitions()` throwing one.